src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -29,9 +33,18 @@
     }
 
     .buttons {
+      visibility: hidden;
```

Comment:
visibility: hidden; jest chyba zbędne jak i tak jest niżej opacity: 0;

Reply:
visibility hidden działa trochę inaczej niż opacity i dlatego wolałem go  użyć, a jako, że jest nieanimowalne to dodałem opacity, żeby działało transition